docs_source_files/content/legacy_docs/selenium_rc.es.md

```diff
@@ -842,112 +842,133 @@ En segundo plano, el navegador realmente realizará una operación *type*,
-varying data that you will require programming logic for verifying expected 
-results.  Basically, the Selenium-IDE does not support iteration and 
-standard condition statements.  You can do some conditions by embedding javascript
-in Selenese parameters, however 
-iteration is impossible, and most conditions will be much easier in a  
-programming language.  In addition, you may need exception handling for
-error recovery.  For these reasons and others, we have written this section
-to illustrate the use of common programming techniques to
-give you greater 'verification power' in your automated testing.
-
-The examples in this section are written
-in C# and Java, although the code is simple and can be easily adapted to the other supported
-languages.  If you have some basic knowledge
-of an object-oriented programming language you shouldn't have difficulty understanding this section.
-
-### Iteration
-
-Iteration is one of the most common things people need to do in their tests.
-For example, you may want to to execute a search multiple times.  Or, perhaps for
-verifying your test results you need to process a "result set" returned from a database.
-
-Using the same Google search example we used earlier, let's 
-check the Selenium search results. This test could use the Selenese:
+##### Registrar los Comandos Selenese
+
+* Logging Selenium se puede utilizar para generar un reporte de todos los comandos de Selenese en tu prueba junto con el resultado exitoso o no
+de cada uno. Logging Selenium extiende el controlador del cliente Java para agregar esta capacidad de registro de Selenese.
+
+#### Reportes de Pruebas para Python
+
+* Al utilizar Python Client Driver, HTMLTestRunner se puede utilizar para
+generar un reporte de prueba.
+
+#### Reportes de Pruebas para Ruby
+
+* Si el framework RSpec se utiliza para escribir los casos de prueba de
+Selenium en Ruby, entonces su reporte HTML se puede utilizar para generar
+un reporte de prueba.
+
+## Agregando Sabor a Tus Pruebas
+
+Ahora llegaremos a la razón de por que utilizar Selenium RC,
+agregando lógica de programación a tus pruebas. Es lo mismo que
+para cualquier programa.
+El flujo del programa se controla mediante declaraciones de condición e iteración.
+Además, puedes reportar la información de progreso mediante I/O.
+En esta sección mostraremos algunos ejemplos de cómo las construcciones
+del lenguaje de programación se pueden combinar con Selenium para
+resolver problemas comunes en las pruebas.
+
+Encontrarás que a medida que transicionas de pruebas simples
+como la existencia de elementos en una página, a pruebas de funcionalidad
+dinámica que involucran múltiples páginas web y datos variables, que
+vas a necesitar la lógica de programación para verificar los resultados
+esperados.
+Básicamente, Selenium-IDE no soporta declaraciones estándar de iteracion
+y de condición.
+Puede hacer algunas condiciones incrustando javascript en los parámetros
```

Comment:
```suggestion
Puede hacer algunas condiciones incrustando JavaScript en los parámetros
```

Reply:
This is actually like it is in the rest of the docs, will update in all langs instead of only this one, one sec.